.github/actions/argus-builder/build-prep/src/main.test.ts

```diff
@@ -0,0 +1,331 @@
+import {
+  findMatchingChangedFiles, isLabelOnPullRequest, isMatchingBranch, ProcessedImage, processImagesInput, wildcardMatch,
+} from './main';
+
+// mock core to suppress logs
+jest.mock('@actions/core');
+
+describe('validate-json-schema', () => {
+  describe('wildcardMatch', () => {
```

Comment:
I wonder if anyone uses special characters in their label. will labels that use regex symbols trip up the function? eg `$myvariable` or `build[amd64]`

Reply:
Made it work with special characters in [5ef9f21](https://github.com/chanzuckerberg/github-actions/pull/386/commits/5ef9f210f6cd8849e7e8ea63556e1f50536468df) and added tests there to show that it works

Reply:
Good callout